src/allmydata/crypto/aes.py

```diff
@@ -0,0 +1,100 @@
+"""
+Helper functions for cryptograhpy-related operations inside Tahoe
+using AES
+
+These functions use and return objects that are documented in the
+`cryptography` library -- however, code inside Tahoe should only use
+functions from allmydata.crypto.aes and not rely on features of any
+objects that `cryptography` documents.
+"""
+
+import six
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.ciphers import (
+    Cipher,
+    algorithms,
+    modes,
+    CipherContext,
+)
+
+
+DEFAULT_IV = b'\x00' * 16
```

Comment:
I'm mildly surprised that `b'\x00' * 16` is 16 nuls on Python 3 and not, say, `b'\x00'`.

Reply:
Not sure what you mean?